### DISCUSSION TITLE
Unable to display pandas dataframe using perspective python module

### DISCUSSION THREAD

**Discussion by ghouse41** [perspective-dev/perspective#1474] [Q&A] [answered]
Unable to see perspective widget ui in jupyter lab. 

I’m trying to display pandas dataframe in perspective widget. But unable to do that.

jupyter lab version - 3.0.16
Also installed the jupyter lab extension 

Can anyone help me here. How to display dataframe using perspective widget.

![image](https://user-images.githubusercontent.com/55399957/124966622-26009300-e041-11eb-9e6d-8c26ba1519d2.jpg)

**Comment by timkpaine** [comment, marked as answer]
after installing `@jupyter-widgets/jupyterlab-manager` and `@finos/perspective-jupyterlab`, did you refresh the page? Are there any console errors in the browser (open the developer tools or hit F12 and see if there are red errors)

**Reply by ghouse41** [reply]
It worked after refreshing thank you.
